README.md

```diff
@@ -6,36 +6,24 @@ By default, Split’s SDKs keep segment and split data synchronized as users nav
 
 [![Twitter Follow](https://img.shields.io/twitter/follow/splitsoftware.svg?style=social&label=Follow&maxAge=1529000)](https://twitter.com/intent/follow?screen_name=splitsoftware)
 
-## Compatibility
-Split Synchronizer supports Go version 1.8 or higher.
-
-## Getting started
-Below is a simple example that describes the instantiation of Split Synchronizer:
-
-### Usage via Go
-1. Install dependencies via `dep ensure`
-2. Then, execute `go run main.go -api-key "<YOUR_SDK_TYPE_API_KEY>" -redis-host "<YOUR_REDIS_HOST>" -redis-port <YOUR_REDIS_PORT> -redis-prefix "<YOUR_PREFIX>"`
-
-### Docker
-1. You can pull the Docker image from [Docker Hub](https://hub.docker.com/r/splitsoftware/split-synchronizer) and run it into your container environment.
+## Deploying to Heroku
+1. Create a Heroku app by going to the [private space](https://dashboard.heroku.com/teams/states-title/spaces) you want to add the new synchronizer to
+2. Set the image as container via 
+> `heroku stack:set container -a <app_name>`
+3. Add the go language pack via 
+> `heroku buildpacks:set heroku/go -a <app_name>`
```

Comment:
@fsaresh For docker images, we shouldn't need to configure buildpacks since the image has all build artifacts. The instructions here looks like it's more for a regular Heroku app deployment than a container deployment.

Reply:
You're right. This also made me realize I could use the `heroku create` CLI command to take care of most of this for us anyway. It reduced the number of instructions and the amount of non-programmatic effort pretty significantly.